### DISCUSSION TITLE
Is running on port 443 required?

### DISCUSSION THREAD

**Discussion by egasvegas1109** [XTLS/Xray-core#3636] [Q&A]
Hello, is it necessary to run xray-core with VLESS protocol on port 443? If not, what do I lose if I run it on a different port? I use nginx redirection on xray.

**Comment by donnyxray** [comment]
We do not use port 443. You don't lose anything.

The only problem you may run into occasionally is that some hotels block all ports except 80/443/993/995/465/etc.

**Reply by egasvegas1109** [reply]
Thanks, but VLESS is masquerading as https. Won't it be suspicious from the outside if the traffic doesn't come from 443 projects?
